push.js

```diff
@@ -101,9 +102,11 @@ for (const block of blocks) {
   embed.setColor(0x991d12)
 
   let counter = 0
+  var turndownService = new turndown;
   // with all section points auto-incrementing
   for (const row of block.points) {
-    embed.addField(`${++counter}.`, row.innerHTML)
+    var row_to_add = turndownService.turndown(row.innerHTML)
```

Comment:
Use `const` instead of `var`, use camelCase instead of snake_case

Reply:
@kpostekk fixed nits